routes/api.php

```diff
@@ -1 +1,9 @@
-<?php
\ No newline at end of file
+<?php
+
+Route::get('me', 'ApiController@me')
+    ->name('api.me')
+    ->middleware('auth:api');
+
+Route::get('users', 'ApiController@users')
+    ->name('api.users')
+    ->middleware('client:list-users');
```

Comment:
don't you need a [middleware](https://laravel.com/docs/5.8/passport#checking-scopes) to check the scope?